### ISSUE TITLE
Improve guidance on alt text in tweets

### ISSUE THREAD

**Issue by matteason** [GSA/digitalgov.gov#4838]
**Improving the Accessibility of Social Media in Government**
https://digital.gov/resources/improving-the-accessibility-of-social-media-in-government/#tweets

---

Hello from the UK! 👋 

Your guidance on making tweets accessible says:

> If your tweet links to photo, video or audio content, make your tweet act as a descriptive caption so it provides context for the item, and then link back to a website page that hosts a tagged photo, captioned video or audio with full caption.
> [...]
> Ensure that you link to accessible content, i.e., a tagged photo, captioned video or audio with written transcript.
>
> - A tagged photo simply has alternative text associated with it that describes the image. For more details on alternative text, please visit [http://webaim.org/articles/gonewild/#alttext.](http://webaim.org/articles/gonewild/#alttext)

I think this guidance could be improved by saying that all images should have alt text added in the tweet iself, rather than linking to an accessible version externally. Twitter allows you to add alt text directly to images posted on Twitter (via 'Add description' in the tweet compose modal), and this is preferred by blind and visually impaired users because it doesn't mean they have to leave Twitter to find out what's in the image.

Twitter recently added an ALT badge to all images with alt text, allowing sighted users to see alt text too, which is raising awareness of alt text in tweets: https://twitter.com/TwitterA11y/status/1512191972856975364

In the UK our Government Communication Service provides [this guidance on alt text](https://gcs.civilservice.gov.uk/guidance/digital-communication/planning-creating-and-publishing-accessible-social-media-campaigns/#Accessibility-best-practice-for-community-managers-and-publishers), which includes instructions on how to add it on FB, Instagram and Twitter.

(The context for this is that I've just launched [a Twitter bot](https://twitter.com/UKGovAltBot) which flags up when UK government departments post images without alt text. I've had a request for a US version so I was looking for a authoritative Government source for social media managers which says why and how to add alt text and found the page on digital.gov)

**Comment by matteason** [comment]
Commenting to avoid auto-close as the issue is still there and I can't remove the label

**Comment by matteason** [comment]
Sorry, closed in error. This issue is still present

**Comment by matteason** [comment]
Sorry, closed in error. This issue is still present
